vtex-apps/reviews-and-ratings
# Use correct product URL as schema id

#### What problem is this solving?
The current schema generated for products uses the full URL - using portal.vtexcommercestable.com.br - as its @id. This PR ensures that `v3.x` has the same pattern as `v2.x` ([PR here](https://github.com/vtex-apps/reviews-and-ratings/pull/116)) and the `Structured Data` app ([PR here](https://github.com/vtex-apps/structured-data/pull/47)).



#### How to test it?



[This workspace](https://schemaorgid--bladehq.myvtex.com/item--real-steel-compact-g5-metamorph--134404/p) has the branch linked. It's possible to check that the object generated as schema has the following structure:

```
{
  "@context": "http://schema.org",
  "@type": "Product",
  "@id": "https://schemaorgid--bladehq.myvtex.com/item--real-steel-compact-g5-metamorph--134404/p",
  "aggregateRating": {
    "@type": "AggregateRating",
    "ratingValue": "5",
    "reviewCount": "3"
  },
  "name": "Real Steel Compact G5 Metamorph  Knife Marble CF (3.0\" M390) Exclusive"
}
```

[The live version](https://bladehq.myvtex.com/item--real-steel-compact-g5-metamorph--134404/p) has the following structure:

```
{
  "@context": "http://schema.org",
  "@type": "Product",
  "@id": "https://portal.vtexcommercestable.com.br/item--real-steel-compact-g5-metamorph--134404/p",
  "aggregateRating": {
    "@type": "AggregateRating",
    "ratingValue": "5",
    "reviewCount": "3"
  },
  "name": "Real Steel Compact G5 Metamorph  Knife Marble CF (3.0\" M390) Exclusive"
}
```
